fields/types/password/PasswordType.js

```diff
@@ -133,6 +150,17 @@ password.prototype.validateInput = function (data, callback) {
 		detail = 'passwords must match';
 	}
 	// TODO: we could support a password complexity option (or regexp) here
+	if (min && typeof passwordValue === 'string') {
+		result = passwordValue.length > min;
+	}
+	for (var prop in complexity) {
+		if (complexity[prop] && typeof passwordValue === 'string') {
+			result = (regexChunk[prop]).test(passwordValue);
```

Comment:
There needs to be a check before here that makes sure that `regexChunk[prop]` exists, and if not console.logging the fact that the `complexity` option was configured wrongly on field with name `"whatever"`.